x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/epm/screens/home/index.tsx

```diff
@@ -4,136 +4,145 @@
 import { WithHeaderLayout } from '../../../../layouts';
-import { CategorySummaryItem, PackageList } from '../../../../types';
+import { CategorySummaryItem } from '../../../../types';
 import { PackageListGrid } from '../../components/package_list_grid';
-// import { useBreadcrumbs, useLinks } from '../../hooks';
 import { CategoryFacets } from './category_facets';
 import { HeroCopy, HeroImage } from './header';
-import { useAllPackages, useInstalledPackages, useLocalSearch } from './hooks';
-import { SearchPackages } from './search_packages';
 
-export function Home() {
-  // useBreadcrumbs([{ text: PLUGIN.TITLE, href: toListView() }]);
+export function EPMHomePage() {
+  const {
+    params: { tabId },
+  } = useRouteMatch<{ tabId?: string }>();
 
-  const state = useHomeState();
-  const searchBar = (
-    <EuiSearchBar
-      query={state.searchTerm}
-      key="search-input"
-      box={{
-        placeholder: 'Find a new package, or one you already use.',
-        incremental: true,
-      }}
-      onChange={({ queryText: userInput }: { queryText: string }) => {
-        state.setSearchTerm(userInput);
-      }}
-    />
-  );
-  const body = state.searchTerm ? (
-    <SearchPackages
-      searchTerm={state.searchTerm}
-      localSearchRef={state.localSearchRef}
-      allPackages={state.allPackages}
-    />
-  ) : (
-    <Fragment>
-      {state.installedPackages.length ? (
-        <Fragment>
-          <InstalledPackages list={state.installedPackages} />
-          <EuiHorizontalRule margin="xxl" />
-        </Fragment>
-      ) : null}
-      <AvailablePackages {...state} />
-    </Fragment>
-  );
+  const ALL_PACKAGES_URI = useLink(EPM_LIST_ALL_PACKAGES_PATH);
+  const INSTALLED_PACKAGES_URI = useLink(EPM_LIST_INSTALLED_PACKAGES_PATH);
 
   return (
     <WithHeaderLayout
       leftColumn={<HeroCopy />}
       rightColumn={<HeroImage />}
-      // tabs={[
-      //   {
-      //     id: 'all_packages',
-      //     name: 'All packages',
-      //     isSelected: true,
-      //   },
-      //   {
-      //     id: 'installed_packages',
-      //     name: 'Installed packages',
-      //   },
-      // ]}
+      tabs={
+        ([
+          {
+            id: 'all_packages',
+            name: i18n.translate('xpack.ingestManager.epmList.allPackagesTabText', {
+              defaultMessage: 'All packages',
+            }),
+            href: ALL_PACKAGES_URI,
+            isSelected: !tabId,
+          },
+          {
+            id: 'installed_packages',
+            name: i18n.translate('xpack.ingestManager.epmList.installedPackagesTabText', {
+              defaultMessage: 'Installed packages',
+            }),
+            href: INSTALLED_PACKAGES_URI,
+            isSelected: tabId === 'installed',
+          },
+        ] as unknown) as EuiTabProps[]
+      }
     >
-      {searchBar}
-      <EuiSpacer size="m" />
-      {body}
+      {tabId === 'installed' ? <InstalledPackages /> : <AvailablePackages />}
```

Comment:
@jen-huang can we use react-router sub routing here instead of a custom one, we can have a proper 404 handling if user type a wrong tab, @paul-tavares did this for the agent config detail tabs